### ISSUE TITLE
error in Window-Classifier-for-NER model

### ISSUE THREAD

**Issue by airship-explorer** [DSKSD/cs-224n-Pytorch#4]
In the 4th jupyter file named `Word Window Classification and Neural Networks`, I found something wrong.  Specifically, in the class `WindowClassifier`, you has used `self.softmax = nn.LogSoftmax(dim=1)`in the output layer, then you don't need use `CrossEntropyLoss()` loss, you must use `torch.nn.NLLLoss` loss
